tools/setup_helpers/extension.py

```diff
@@ -100,11 +99,6 @@ def build_extension(self, ext):
 
         build_args = ["--config", cfg]
 
-        if _BUILD_S3 and _AWSSDK_DIR:
-            cmake_args += [
-                f"-DAWSSDK_DIR={_AWSSDK_DIR}",
-            ]
```

Comment:
Removing `AWSSDK_DIR` as it's redundant now because it's compiled in third-party library. And, users can use system-wise `AWSSDK` if needed with environment variable `USE_SYSTEM_AWS_SDK_CPP`